### ISSUE TITLE
Example of how to use Vyatta floppy disk images created with the `vyatta` command?

### ISSUE THREAD

**Issue by activeshadow** [sandia-minimega/minimega#267]
I can't for the life of me figure out how to get VyOS to use the `/config/config.boot` present in the floppy image created using the `vyatta` command when booting an instance of it with **minimega**. I've searched the Googles and tried a few variations of the `-drive` option via the `qemu-append` config directive but to no avail.

If someone can point me in the right direction I can submit a pull request with updated documentation that includes an example of how to use the floppy image created with the `vyatta` command.

**Comment by bjwrigh** [comment]
Without being completely familiar with VyOS, only the last release of open
source Vyatta, you can/should be able to inject the config into the correct
location for it to load on boot.  In Vyatta, its in
/opt/somewhere/etc/config/config.boot.  See the Minimega help for VM
inject, and find the stored location of the config file on the VyOS disk.

-B

**Comment by activeshadow** [comment]
Thanks @bjwrigh. I knew I could go about it with `vm inject`, but that would involve the actual `config.boot` file that ends up residing in the floppy image generated by the `vyatta` command. I assume there's a way to *attach* the floppy to the VM in such a way that the running instance of Vyatta/VyOS uses it, otherwise why have a command that generates a floppy image?

At this point, it's just my curiosity about getting the floppy image to work that's driving me to antagonize everyone with this issue... :wink:

**Comment by djfritz** [comment]
We've been attaching the floppy image with `-fda` and the qemu-append API:

    # vyatta config
    vyatta ospf... etc.
    
    vyatta write foo
    vm config qemu-append -fda /path/to/foo

**Comment by djfritz** [comment]
This does need some documentation.

**Comment by djfritz** [comment]
Merging this into #268, though github doesn't allow merging issues, so we'll do the best we can:

Duplicate of #268

**Comment by activeshadow** [comment]
Ah... `-fda` works eh? I should have known to try that. Thx!

**Comment by activeshadow** [comment]
FYI, it doesn't look like this works with VyOS... at least it didn't for me anyway. I tried it both with a LiveCD (ie. `vm config cdrom ...`) and with a disk image I had installed VyOS to.

This blog post also eludes to the floppy approach no longer being supported.

http://blog.vyos.net/post/65961831033/new-images-and-git-digest-3

I'm going to close this issue since the main request was for documentation on how to use the floppy image and a separate issue has been opened for that specifically.
